app/admin/cclow_page.rb

```diff
@@ -0,0 +1,67 @@
+ActiveAdmin.register CCLOWPage do
```

Comment:
I think if this is the same, just menu is different and class name we could unified it in one file, the same as we do with Tags. Could be done later.

Reply:
let's leave it for later